lib/utils.py

```diff
@@ -92,3 +92,40 @@ def inline_anova(anova,
 		if pythontex_safe:
 			inline = inline.replace("\\","\\\\")
 	return inline
+
+def bold_max_row_value(df, df_tex: str):
+    """
+    Wraps the maximum value for each row with \textbf{} in a df.to_latex() string.
+
+    Parameters
+    ----------
+    df :  pandas.DataFrame
+            Pandas DataFrame object containing at least one column with numerical values.
+
+    df_tex: string
+                The output of df.to_latex(). The values if this string will be changed such that the maximum values of
+                each row will be bold in the resulting pdf.
+    >>> import pandas as pd
```

Comment:
very very good. Absolutely awesome that you added a usage example! so many ppl including me don't do this and that sucks :(

Reply:
yes it's very useful, especially if the documentation is not clear haha